[Intel GPU] Update xpu triton commit pin for PyTorch release 2.8.

etaf opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #154194

As title.
Thanks @anmyachev  for the work on compatibility adaptation.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov

etaf committed:
[Intel GPU] Update xpu triton commit pin for PyTorch release 2.8.
Compared to the current version of PyTorch's OpenAI Triton, the updated
XPU Triton is based on a newer version of OpenAI Triton. In terms of
integration interface, this update includes two main changes:

1. setup.py has been moved from the triton/python directory to the root
triton/ directory.

2. The CMake version requirement has been updated to cmake>=3.20,<4.0.

[ghstack-poisoned]

etaf committed:
Update on "[WIP][Intel GPU] Update xpu triton commit pin for PyTorch release 2.8."


Compared to the current version of PyTorch's OpenAI Triton, the updated
XPU Triton is based on a newer version of OpenAI Triton. In terms of
integration interface, this update includes two main changes:
1. setup.py has been moved from the `TRITON_ROOT/python` directory to the `TRITON_ROOT` directory.
2. The CMake version requirement has been updated to cmake>=3.20,<4.0.

Thanks anmyachev  for the work on compatibility adaptation.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

etaf committed:
Update on "[WIP][Intel GPU] Update xpu triton commit pin for PyTorch release 2.8."


Compared to the current version of PyTorch's OpenAI Triton, the updated
XPU Triton is based on a newer version of OpenAI Triton. In terms of
integration interface, this update includes two main changes:
1. setup.py has been moved from the `TRITON_ROOT/python` directory to the `TRITON_ROOT` directory.
2. The CMake version requirement has been updated to cmake>=3.20,<4.0.

Thanks anmyachev  for the work on compatibility adaptation.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

etaf committed:
Update on "[WIP][Intel GPU] Update xpu triton commit pin for PyTorch release 2.8."


Compared to the current version of PyTorch's OpenAI Triton, the updated
XPU Triton is based on a newer version of OpenAI Triton. In terms of
integration interface, this update includes two main changes:
1. setup.py has been moved from the `TRITON_ROOT/python` directory to the `TRITON_ROOT` directory.
2. The CMake version requirement has been updated to cmake>=3.20,<4.0.

Thanks anmyachev  for the work on compatibility adaptation.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

etaf committed:
Update on "[WIP][Intel GPU] Update xpu triton commit pin for PyTorch release 2.8."


Compared to the current version of PyTorch's OpenAI Triton, the updated
XPU Triton is based on a newer version of OpenAI Triton. In terms of
integration interface, this update includes two main changes:
1. setup.py has been moved from the `TRITON_ROOT/python` directory to the `TRITON_ROOT` directory.
2. The CMake version requirement has been updated to cmake>=3.20,<4.0.

Thanks anmyachev  for the work on compatibility adaptation.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

etaf committed:
Update on "[WIP][Intel GPU] Update xpu triton commit pin for PyTorch release 2.8."


Compared to the current version of PyTorch's OpenAI Triton, the updated
XPU Triton is based on a newer version of OpenAI Triton. In terms of
integration interface, this update includes two main changes:
1. setup.py has been moved from the `TRITON_ROOT/python` directory to the `TRITON_ROOT` directory.
2. The CMake version requirement has been updated to cmake>=3.20,<4.0.

Thanks anmyachev  for the work on compatibility adaptation.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

etaf committed:
Update on "[WIP][Intel GPU] Update xpu triton commit pin for PyTorch release 2.8."


Compared to the current version of PyTorch's OpenAI Triton, the updated
XPU Triton is based on a newer version of OpenAI Triton. In terms of
integration interface, this update includes two main changes:
1. setup.py has been moved from the `TRITON_ROOT/python` directory to the `TRITON_ROOT` directory.
2. The CMake version requirement has been updated to cmake>=3.20,<4.0.

Thanks anmyachev  for the work on compatibility adaptation.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

etaf committed:
Update on "[WIP][Intel GPU] Update xpu triton commit pin for PyTorch release 2.8."


Compared to the current version of PyTorch's OpenAI Triton, the updated
XPU Triton is based on a newer version of OpenAI Triton. In terms of
integration interface, this update includes two main changes:
1. setup.py has been moved from the `TRITON_ROOT/python` directory to the `TRITON_ROOT` directory.
2. The CMake version requirement has been updated to cmake>=3.20,<4.0.

Thanks anmyachev  for the work on compatibility adaptation.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

etaf committed:
Update on "[WIP][Intel GPU] Update xpu triton commit pin for PyTorch release 2.8."


Compared to the current version of PyTorch's OpenAI Triton, the updated
XPU Triton is based on a newer version of OpenAI Triton. In terms of
integration interface, this update includes two main changes:
1. setup.py has been moved from the `TRITON_ROOT/python` directory to the `TRITON_ROOT` directory.
2. The CMake version requirement has been updated to cmake>=3.20,<4.0.

Thanks anmyachev  for the work on compatibility adaptation.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

etaf committed:
Update on "[WIP][Intel GPU] Update xpu triton commit pin for PyTorch release 2.8."


Compared to the current version of PyTorch's OpenAI Triton, the updated
XPU Triton is based on a newer version of OpenAI Triton. In terms of
integration interface, this update includes two main changes:
1. setup.py has been moved from the `TRITON_ROOT/python` directory to the `TRITON_ROOT` directory.
2. The CMake version requirement has been updated to cmake>=3.20,<4.0.

Thanks anmyachev  for the work on compatibility adaptation.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

etaf committed:
Update on "[WIP][Intel GPU] Update xpu triton commit pin for PyTorch release 2.8."


As title.
Thanks anmyachev  for the work on compatibility adaptation.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

etaf committed:
Update on "[WIP][Intel GPU] Update xpu triton commit pin for PyTorch release 2.8."


As title.
Thanks anmyachev  for the work on compatibility adaptation.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

chuanqi129 commented:
@etaf please rebase this PR and change the triton_xpu_version to 3.4 in this PR also

etaf committed:
Update on "[WIP][Intel GPU] Update xpu triton commit pin for PyTorch release 2.8."


As title.
Thanks anmyachev  for the work on compatibility adaptation.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

etaf committed:
Update on "[WIP][Intel GPU] Update xpu triton commit pin for PyTorch release 2.8."


As title.
Thanks anmyachev  for the work on compatibility adaptation.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

etaf committed:
Update on "[WIP][Intel GPU] Update xpu triton commit pin for PyTorch release 2.8."


As title.
Thanks anmyachev  for the work on compatibility adaptation.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]
